src/graphics/SVG/Radar/MeshRenderer.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types=1);
+
+/*
+ * (c) Vladimir "allejo" Jimenez <me@allejo.io>
+ *
+ * For the full copyright and license information, please view the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+namespace allejo\bzflag\graphics\SVG\Radar;
+
+use allejo\bzflag\graphics\Common\WorldBoundary;
+use allejo\bzflag\graphics\SVG\ISVGStylable;
+use allejo\bzflag\graphics\SVG\Radar\Styles\MeshStyle;
+use allejo\bzflag\graphics\SVG\SVGStylableUtilities;
+use SVG\Nodes\Structures\SVGGroup;
+use SVG\Nodes\SVGNode;
+
+/**
+ * @internal
+ *
+ * @extends ObstacleRenderer<\allejo\bzflag\world\Object\MeshObstacle>
+ * @implements ISVGStylable<\allejo\bzflag\world\Object\MeshObstacle>
+ */
+class MeshRenderer extends ObstacleRenderer implements ISVGStylable
+{
+    /** @var MeshStyle */
+    public static $STYLE;
+
+    /** @var MeshObstacle */
+    protected $obstacle;
+
+    /**
+     * @param MeshObstacle $mesh
+     */
+    public function __construct($mesh, WorldBoundary $worldBoundary)
+    {
+        if (self::$STYLE === null)
+        {
+            self::$STYLE = new MeshStyle();
+        }
+
+        parent::__construct($mesh, $worldBoundary);
+    }
+
+    public function exportSVG(): SVGNode
+    {
+        $svg = $this->meshToSVGNode(SVGGroup::class);
+
+        self::stylizeSVGNode($svg, $this->obstacle);
+
+        if ($this->bzwAttributesEnabled)
+        {
+            self::attachBzwAttributes($svg, $this->obstacle);
+        }
+
+        return $svg;
+    }
+
+    /**
+     * @param null|MeshObstacle $obstacle
+     */
+    public static function attachBzwAttributes(SVGNode $node, $obstacle): void
+    {
+        $node->setAttribute('bzw:vertices', json_encode($obstacle->getVertices()));
```

Comment:
```suggestion
        $node->setAttribute('bzw:type', (string)$obstacle->getObjectType());
        $node->setAttribute('bzw:vertices', json_encode($obstacle->getVertices()));
        $node->setAttribute('bzw:faces', json_encode($obstacle->getFaces()));
```

Let's have this function handle the debug attributes